src/components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,149 @@
+import { useState } from 'react';
+import { ErrorMessage } from '../ErrorMessage';
+import { User } from '../../types/User';
+import { addUser, getUser } from '../../api/user';
+
+type Props = {
+  setUser: (user: User) => void;
+};
+
+export const LoginForm: React.FC<Props> = ({ setUser }) => {
+  const [email, setEmail] = useState('');
+  const [name, setName] = useState('');
+  const [error, setError] = useState<string>('');
+  const [wasUserNotFound, setWasUserNotFound] = useState<boolean>(false);
```

Comment:
I suppose the variable name can be simpler 
```suggestion
  const [isUserFound, setIsUserFound] = useState<boolean>(true);
```